Makefile

```diff
@@ -254,7 +264,9 @@ RH_BUNDLE_VERSION?=$(VERSION)
 RH_DEPLOY_PATH="deploy/certified-metadata-bundle"
 RH_DEPLOY_FULL_PATH="$(RH_DEPLOY_PATH)/cockroach-operator/"
 RH_COCKROACH_DATABASE_IMAGE=registry.connect.redhat.com/cockroachdb/cockroach:$(COCKROACH_DATABASE_VERSION)
-RH_OPERATOR_IMAGE?=registry.connect.redhat.com/cockroachdb/cockroachdb-operator:$(APP_VERSION)
+# TODO this is coackroachdb-operator, but we are using coachroach-operator
```

Comment:
@rail here is another thing that was wrong.  Are we overriding cockroachdb -> cockroach during the release?

Reply:
This is actually correct here - in redhat it's 'cockroachdb-operator' in github and dockerhub it's 'cockroach-operator'

Reply:
`registry.connect.redhat.com/cockroachdb/cockroachdb-operator:$(APP_VERSION)` is correct. Not sure if we want to rename it...

Reply:
We should pick one, can dockerhub do redirects?

Reply:
There can only be one! Forgive the movie reference 😂. Can we just get one name please

Reply:
we can't change the Red Hat name without breaking our Red Hat Marketplace listing